volatility/framework/plugins/linux/check_idt.py

```diff
@@ -0,0 +1,94 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
```

Comment:
The year should be the year it's added to the repo, hopefully shortly after it's submitted 5;)...

Reply:
i fixed this in the most recent commit